projects/plugins/jetpack/extensions/blocks/ai-assistant/inline-extensions/lib/is-possible-to-extend-block.ts

```diff
@@ -30,8 +31,11 @@ export function isPossibleToExtendBlock( blockName: string ): boolean {
 		return false;
 	}
 
-	// Check if AI Assistant extensions support is enabled.
-	if ( ! isAiAssistantExtensionsSupportEnabled ) {
+	// Check if AI Assistant extensions support is enabled or the block is in the released list
```

Comment:
Could we just rely on `EXTENDED_INLINE_BLOCKS.includes( blockName )` on this check?

Reply:
Yes, we can. Good catch!